subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1874,8 +1929,8 @@ void ull_conn_done(struct node_rx_event_done *done)
 		ticker_status = ticker_update(TICKER_INSTANCE_ID_CTLR,
 					      TICKER_USER_ID_ULL_HIGH,
 					      ticker_id,
-					      ticks_drift_plus,
-					      ticks_drift_minus, 0, 0,
+					      ticks_drift_plus, ticks_drift_minus,
+					      ticks_slot_plus, ticks_slot_minus,
```

Comment:
In my perspective, updating the slot reservation based on DLE `max_octets` is incorrect for this controller. The ticker shall only reserve the time needed for basic LL transactions, i.e. for a PDU size of 27 bytes in each direction.
Using `max_octets` will cause too heavy reservations for ACL connections, causing substantial scheduling collisions.